src/silx/gui/plot/backends/BackendOpenGL.py

```diff
@@ -590,10 +602,9 @@ def _renderItems(self, overlay=False):
                     continue
 
                 color = item["color"]
-                intensity = color[0] * 0.299 + color[1] * 0.587 + color[2] * 0.114
-                bgColor = (
-                    (1.0, 1.0, 1.0, 0.75) if intensity <= 0.5 else (0.0, 0.0, 0.0, 0.75)
-                )
+                bgColor = item["bgcolor"]
+                if bgColor is None:
+                    bgColor = 1.0, 1.0, 1.0, 1.0
```

Comment:
`GLText` accept `bgColor=None`, so that's not needed, or did I missed something?

```suggestion
```

Reply:
`item["bgcolor"]` could even be passed directly to `glutils.Text2D`